inc/sainexthopgroup.h

```diff
@@ -59,22 +59,40 @@ typedef enum _sai_next_hop_group_attr_t
     /** Number of next hops in the group [uint32_t] */
     SAI_NEXT_HOP_GROUP_ATTR_NEXT_HOP_COUNT,
 
+    /** Next hop list [sai_object_list_t] */
+    SAI_NEXT_HOP_GROUP_ATTR_NEXT_HOP_MEMBER_LIST,
+
     /** READ-WRITE */
 
     /** Next hop group type [sai_next_hop_group_type_t] (MANDATORY_ON_CREATE|CREATE_ONLY) */
     SAI_NEXT_HOP_GROUP_ATTR_TYPE,
 
-    /** Next hop list [sai_object_list_t] (MAXDATORY_ON_CREATE) 
-     * The next hop group must have at least one next hop member at the creation time */
-    SAI_NEXT_HOP_GROUP_ATTR_NEXT_HOP_LIST,
```

Comment:
@itaibaz, @kcudnik, one thing here is that we'll have create_switch API, where a few defaults objects are created along with the switch object. In this sense, creating next hop members in a next hop group call is not different from creating default hash object during the switch creation.